ext/bcmath/bcmath.stub.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+function bcadd(string $left_operand, string $right_operand, int $scale = 0) : string {}
```

Comment:
Let's make these `$scale` defaults `= UNKNOWN`, because the actual default scale is decided by an internal global (generally not 0).

Reply:
Kk, bit confusing the constant does not exists. Got it tho.